api/lib/routes/routeHelper.js

```diff
@@ -13,19 +13,32 @@ exports.isString = function(value) {
   return typeof(value) === "string";
 }
 
-exports.pagination = function(resultList, page, resultsPerPage) {
+exports.pagination = function(resultList, page, resultsPerPage, req) {
   var result = {};
   var totalResults = resultList.length;
   var totalPages = 0;
   var perPage = Number.parseInt(resultsPerPage);
   var pageNo = Number.parseInt(page);
-  totalPages = (totalResults % perPage) === 0 ? (totalResults / perPage) : (totalResults / perPage + 1);
+  var prevUrl = null;
+  var nextUrl = null;
+  totalPages = (totalResults % perPage) === 0 ? (totalResults / perPage) : (Number.parseInt(totalResults / perPage) + 1);
```

Comment:
why Number.parseInt ?

Reply:
var totalResults = 5;
var resultsPerPage = 2;
var totalPages = 0;
var perPage = Number.parseInt(resultsPerPage);
totalPages = (totalResults % perPage) === 0 ? (totalResults / perPage) : (totalResults / perPage + 1);
//totalPages = (totalResults % perPage) === 0 ? (totalResults / perPage) : (Number.parseInt(totalResults / perPage) + 1);
console.log(totalPages)
Without Number.parseInt the result of the code above is 3.5 and after adding it the result is 3.

Reply:
thanks for the explanation.